src/Resources/config/routing/api/post.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing https://symfony.com/schema/routing/routing-1.0.xsd">
+    <route id="get_posts" path="/posts.{_format}" methods="GET" controller="Sonata\NewsBundle\Controller\Api\PostController::getPostsAction" format="json">
+        <requirement key="_format">json|xml|html</requirement>
+    </route>
+    <route id="get_post" path="/posts/{id}.{_format}" methods="GET" controller="Sonata\NewsBundle\Controller\Api\PostController::getPostAction" format="json">
```

Comment:
IMO, we could sort these routes by their ids.

Reply:
I check it. Better keep it in this way becouse order in config is equal to order in `nelmio doc`. Its look much better when we keep this order.